java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -3949,6 +3949,20 @@ private Table getExpectedFileTable() {
         .build();
   }
 
+  private Table getExpectedFileTableWithDecimals() {
+    return new TestBuilder()
+        .column(true, false, false, true, false)
+        .column(5, 1, 0, 2, 7)
+        .column(new Byte[]{2, 3, 4, 5, 9})
+        .column(3l, 9l, 4l, 2l, 20l)
+        .column("this", "is", "a", "test", "string")
+        .column(1.0f, 3.5f, 5.9f, 7.1f, 9.8f)
+        .column(5.0d, 9.5d, 0.9d, 7.23d, 2.8d)
+        .decimal32Column(3, 298, 2473, 2119, 1273, 9879)
+        .decimal64Column(4, 398l, 1322l, 983237l, 99872l, 21337l)
+        .build();
+  }
+
   @Test
   void testParquetWriteToFileNoNames() throws IOException {
```

Comment:
This test and `testParquetWriteToBufferChunked` don't use the new decimals table.  Should those be updated to use this as well?

Reply:
I wanted to leave a few tests that tested the DEFAULT options. I can change them to use the Decimal columns if you want